frontend/src/components/History/index.jsx

```diff
@@ -72,6 +21,86 @@ class History extends React.Component {
         this.cardID, this.rowID, this.label;
     }
 
+    COLUMNS() {
+        return [
+            {
+                Header: "edit",
+                accessor: "edit",
+                show: false
+            },
+            {
+                Header: "id",
+                accessor: "id",
+                show: false
+            },
+            {
+                Header: "hidden",
+                accessor: "metadata",
+                show: false
+            },
+            {
+                Header: "Data",
+                accessor: "data",
+                filterMethod: (filter, row) => {
+                    if (
+                        String(row.data)
+                            .toLowerCase()
+                            .includes(filter.value.toLowerCase())
+                    ) {
+                        return true;
+                    } else {
+                        return false;
+                    }
+                }
+            },
+            {
+                Header: "Old Label",
+                accessor: "old_label",
+                width: 100
+            },
+            {
+                Header: "User",
+                accessor: "profile",
+                width: 100
+            },
+            {
+                Header: "Old Label ID",
+                accessor: "old_label_id",
+                show: false
+            },
+            {
+                Header: "Date/Time",
+                accessor: "timestamp",
+                id: "timestamp",
+                width: 150
+            },
+            {
+                Header: "Verified",
+                width: 80,
+                accessor: "verified",
+                Cell: props => {
+                    if (props.value == "Yes") {
+                        return (<p>Yes</p>);
+                    } else if (props.value != "No") {
+                        return (<p>{props.value}</p>);
+                    } else {
+                        return (<Button variant="success" value={props.row.id} onClick={this.verifyLabel}>Verify</Button>);
```

Comment:
Perhaps I don't understand, but when I verify pre-loaded data, I would expect to see if add to the count of fully verified data on the Details page. Its not for me, but is that expected behavior?

Reply:
It should be! Let me check

Reply:
Good catch! Yes the logic hand't been fixed after the update. I pushed a change now

Reply:
Yes, things look correct now!